catalog/core/catalog-core-definitionparser/src/main/java/ddf/catalog/definition/impl/DefinitionParser.java

```diff
@@ -633,11 +641,46 @@ private ValidatorWrapper getValidator(String key, Outer.Validator validator) {
           break;
         }
       default:
-        throw new IllegalStateException("Validator does not exist. (" + validator.validator + ")");
+        {
+          String[] validators = validator.validator.split("::");
+          if (validators.length != 2) {
+            throw new IllegalStateException(
+                "Validator should be in format of 'validatorId::validatorType', not '"
```

Comment:
❓What is `validatorType` used for?

Reply:
It's there to infer which validator type an attribute should be, in order to cast to a validator type. But since it's only supporting `AttributeValidator` I could just take that out.

Reply:
I think it's actually better to just add a new case, maybe `downstream`, and add what's currently in the default case logic in the `downstream` case, and just leave `default` stay as it was.

Reply:
That sounds like a good solution.

Reply:
And.. I just updated it.